src/signin.js

```diff
@@ -1,101 +1,133 @@
   }
   //올바른 값으로 수정
   else if (validateEmail(emailInput.value)) {
-    if (emailInput.parentElement.querySelector('.errorMessage')) {
-      emailInput.parentElement.querySelector('.errorMessage').remove();
+    if (findClassNameErrorMessage) {
+      findClassNameErrorMessage.remove();
       emailInput.style.borderColor = '#CCD5E3'
     }
   }
-}
+})
 
 //비밀번호 에러 메세지 출력
-function passErrorMessage (e) {
-  //에러메세지 - 값이 없을 경우
-  if(passInput.value.trim() === '') {
-    errorMessage(passInput, '비밀번호를 입력해주세요.');
+const passInput = document.querySelector('#pass');
+passInput.addEventListener('focusout', (e) => {
+  const passwordEmpty = passInput.value.trim() === '';
+  const findClassNameErrorMessage = passInput.parentElement.querySelector('.errorMessage');
+  
+  if(passwordEmpty) {
+    setErrorMessageToElement (passInput, '비밀번호를 입력해주세요.');
   }
   //올바른 값으로 수정
   else if (validatePass(passInput.value)) {
-    if (passInput.parentElement.parentElement.querySelector('.errorMessage')) {
-      passInput.parentElement.parentElement.querySelector('.errorMessage').remove();
+    if (findClassNameErrorMessage) {
+      findClassNameErrorMessage.remove();
       passInput.style.borderColor = '#CCD5E3'
     }
   }
-}
-
-//로그인 시도
-function submitForm (e) {
-  //현재페이지에서 테스트
-  e.preventDefault();
-
-  if (emailInput.value === 'test@codeit.com' && passInput.value === 'codeit101') {
-    window.location.href = "./folder";
-  } else {
-    errorMessage(emailInput, '이메일을 확인해주세요.');
-    errorMessage(passInput, '비밀번호를 확인해주세요.');
-  }
-}
+});
 
 //눈 모양 클릭
-function toggleIcon (e) {
-  if (toggle.previousElementSibling.type === 'password') {
-    toggle.previousElementSibling.type = 'text';
+const toggle = document.querySelector('.fa-solid');
+toggle.addEventListener('click', (e) => {
+  const toggleIcon = toggle.previousElementSibling;
+  
+  if (toggleIcon.type === 'password') {
+    toggleIcon.type = 'text';
     toggle.classList.remove('fa-eye-slash');
     toggle.classList.add('fa-eye');
   } else {
-    toggle.previousElementSibling.type = 'password';
+    toggleIcon.type = 'password';
     toggle.classList.remove('fa-eye');
     toggle.classList.add('fa-eye-slash');
   }
+});
+
+//로그인 시도
+// signForm.addEventListener('submit', (e) => {
+  //   //현재페이지에서 테스트
+  //   e.preventDefault();
+  
+  //   const tryCorrectId = emailInput.value === 'test@codeit.com' && passInput.value === 'codeit101'
+  
+  //   if (tryCorrectId) {
+    //     window.location.href = "./folder";
+    //   } else {
+      //     setErrorMessageToElement (emailInput, '이메일을 확인해주세요.');
+//     setErrorMessageToElement (passInput, '비밀번호를 확인해주세요.');
+//   }
+// });
+
+const signForm = document.querySelector('.signin');
+signForm.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const userEmail = emailInput.value;
+  const userPassword = passInput.value;
+
+  try {
+    const response = await fetch ('https://bootcamp-api.codeit.kr/docs/api/sign-in', {
```

Comment:
문서를 보라는 의도가 익숙하지 않고 컨디션이 좋지 않아 이 부분 혼동이 오신것 같습니다.
해당 주소로 접속하여 https://bootcamp-api.codeit.kr/docs API 요청을 어떻게 보내야하는지 확인 한 후 
https://bootcamp-api.codeit.kr/api/sign-in으로 요청을 보내라는 의도입니다.

https://bootcamp-api.codeit.kr/docs에 접속하셔서 좀 내리다보면 `/api/sign-in` 항목이 있는데 이를 펼쳐보세요. 그러면 

`Request body` 부분을 보시면 body에 어떤 데이터를 넣어라라는 뜻으로 해석 할 수 있습니다.
```javascript
{
   email: string,
   password: string
}
이렇게 되어있는것은 email,password 프로퍼티를 가지는 객체를 body에 넣어서 보내라는 뜻입니다. 


```suggestion
    const response = await fetch ('https://bootcamp-api.codeit.kr/api/sign-in', {
```
docs를 지워주세요.